game/static/game/css/auth.css

```
@@ -17,6 +17,242 @@
+    cursor: pointer;
+    transition: all 0.25s ease;
+}
+
+.theme-toggle:hover {
+    background: rgba(240, 192, 64, 0.1);
+    border-color: var(--accent-gold);
+    color: var(--accent-gold);
+}
+.btn-register {
+
+    text-decoration: none;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)